vignettes/vroom.Rmd

```diff
@@ -246,6 +246,16 @@ vroom(
 )
 ```
 
+## Name repair
+
+Often the names of columns in the original dataset are not ideal to work with. `vroom()` uses the same .name_repair argument as tibble, so you can use one of the default name repair strategies or provide a custom function A great approach is to use the [janitor] (http://sfirke.github.io/janitor/) `make_clean_names()` function as the input.
```

Comment:
```suggestion
Often the names of columns in the original dataset are not ideal to work with. `vroom()` uses the same `.name_repair` argument as tibble, so you can use one of the default name repair strategies or provide a custom function. Another great approach is to use the [janitor](http://sfirke.github.io/janitor/) `make_clean_names()` function as the input.
```

Reply:
But maybe it would be to link `janitor::make_clean_names()` directly to its reference page?

Reply:
you meant like this https://github.com/r-lib/vroom/pull/159/commits/fb71963bbd02cdd4cd9947599d1e113345781b5b ?